routes/quizzes.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const router = express.Router();
+const Quiz = require('../models/quiz');
+
+//Getting all questions
+router.get('/', async (req, res) => {
+  const { quizName } = req.query;
+  try {
+    const quizQuestions = await Quiz.find({ name: quizName });
+    res.status(200).json(quizQuestions);
+  } catch (error) {
+    res.status(500).json({ message: error.message });
+  }
+});
+
+//Creating a question
+router.post('/', async (req, res) => {
+  const quiz = new Quiz({
+    name: req.body.name,
+    question: req.body.question,
+  });
```

Comment:
```suggestion
  const { name, question } = req.body;
  const quiz = new Quiz({ name, question });
```
What do you think about writing it this way?

Reply:
I like it, it's elegant! :) 
Done!